<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListConfigurationDlg</class>
 <widget class="QWidget" name="ListConfigurationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="lwActions">
      <item>
       <property name="text">
        <string comment="Display the previously listed commands (go back)">Back</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="Number">One</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="Number">Two</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="Number">Three</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="Number">Four</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="Number">Five</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="Number">Six</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="Number">Seven</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="Number">Eight</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="Advance to the next part of the list">Next</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="Abort the list selection">Cancel</string>
       </property>
      </item>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Triggers:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KEditListBox" name="elbTriggers">
         <property name="buttons">
          <set>KEditListBox::Add|KEditListBox::Remove</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Icon:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cbShowIcon">
         <property name="text">
          <string>Show icon</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KIconButton" name="ibIcon">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>72</height>
          </size>
         </property>
         <property name="icon">
          <string>view-choose</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Visible trigger:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KLineEdit" name="leVisibleTrigger"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
